docs/usage/faq.md

```diff
@@ -167,3 +167,31 @@ Unpickling data is similar to `eval`. On any public-facing server it is a really
 bad idea to unpickle any data sent from the client. For sending data from client
 to server, try some other serialization format like JSON.
 ```
+
+## How can I use a Python function as an event handler and then remove it later?
+
+Note that the most straight forward way of doing this will not work:
+```py
+from js import document
+def f(*args):
+    document.querySelector("h1").innerHTML += "(>.<)"
+
+# Leaks f and doesn't remove the event listener
+document.body.addEventListener('click', f)
+document.body.removeEventListener('click', f)
+```
```

Comment:
Maybe expand what you mean "won't work" in text instead of the comment,

"""
since this will leak f and ...
"""